src/map/zone_entities.cpp

```diff
@@ -171,7 +171,7 @@ void CZoneEntities::FindPartyForMob(CBaseEntity* PEntity)
 
     // force all mobs in a burning circle to link
     ZONETYPE zonetype = m_zone->GetType();
-    bool forceLink = zonetype == ZONETYPE_BATTLEFIELD || zonetype == ZONETYPE_DYNAMIS || PMob->getMobMod(MOBMOD_SUPERLINK);
+    bool forceLink = zonetype == ZONETYPE_DYNAMIS || PMob->getMobMod(MOBMOD_SUPERLINK);
```

Comment:
I wasn't paying attention when this was being talked about in Discord, this doesn't have any far-reaching consequences, does it?

Reply:
It's redundant because all battlefield mobs already superlink. There are, however, 3 exceptions to that superlink rule and this overrides that exception, so the only thing it's doing is preventing the exceptions. It will also cause problems for Limbus, so the conversation earlier was to burn it.